### ISSUE TITLE
Pizza Perfect (WPStoreLocatorSpider)

### ISSUE THREAD

**Issue by CloCkWeRX** [alltheplaces/alltheplaces#9275]
### Brand name

Pizza Perfect

Pizza takeaway chain in South Africa

### Wikidata ID

Q116619227
https://www.wikidata.org/wiki/Q116619227
https://www.wikidata.org/wiki/Special:EntityData/Q116619227.json

### Store finder url(s)

Official Url(s): https://pizzaperfect.co.za/
- [x] I have run the below and if anything in generated, added it.
- [x] I have manually checked for a storefinder
- [x] This is not a manufacturer listing outlets operating under their own brands / this will not create duplicates
`pipenv run scrapy links https://pizzaperfect.co.za/`
`pipenv run scrapy sitemap --pages https://pizzaperfect.co.za/`
`pipenv run scrapy sf --brand-wikidata=Q116619227 https://pizzaperfect.co.za/`
pipenv run scrapy sf https://pizzaperfect.co.za/stores/
```
from locations.storefinders.wp_store_locator import WPStoreLocatorSpider


class PizzaPerfectZASpider(WPStoreLocatorSpider):
    name = "pizza_perfect_za"
    item_attributes = {
        "brand_wikidata": "Q116619227",
        "brand": "Pizza Perfect",
    }
    allowed_domains = [
        "pizzaperfect.co.za",
    ]
    time_format = "%I:M %p"

```

**Comment by CloCkWeRX** [comment]
File "/home/ubuntu/.pyenv/versions/3.11.9/lib/python3.11/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '9:30 AM' does not match format '%I:M %p'
2024-08-04 13:12:01 [scrapy.core.engine] INFO: Closing spider (finished)

**Comment by arrival-spring** [comment]
Fixed by #9591
